modules/packages/apt_get

```diff
@@ -1,10 +1,4 @@
-#!/bin/sh
-''':'
-# First try to run this script with python3, else run with python
-command -v python3 >/dev/null 2>/dev/null  \
-    && exec python3 "$0" "$@" \
-    || exec python  "$0" "$@"
-'''
+#!/var/cfengine/bin/python
```

Comment:
I do not like this change.   This does not work for cfengine installations that uses different directories, eg:
 * /var/lib/cfengine3 (debian/ubuntu)
 * /opt/cfengine

The same comment for the other changes to the python interpreter path. The current situation I just copy the module files and everything works as expected.

Reply:
What if we use $(sys.bindir)/python instead?

Reply:
Yeah, I was thinking about this, but I wanted to change all the package modules to `.in` files.

Reply:
yeah, it does result in more change, but i think that will be more compatible for other builds.